index.js

```diff
@@ -2,16 +2,16 @@
  * External dependencies
  */
 const request = require('request-promise-native');
+const path = require('path');
```

Comment:
I don't want to be too picky, but can you move this line to the "Internal dependencies" section below?

Reply:
Done.
Thanks for noticing it. I'm also obsessive about these kinds of things :D My bad!